lib/rules/multiline-comment-style.js

```diff
@@ -37,6 +67,8 @@ module.exports = {
     create(context) {
         const sourceCode = context.getSourceCode();
         const option = context.options[0] || "starred-block";
+        const params = context.options[1] || {};
+        const allowJSDoc = "allowJSDoc" in params ? params.allowJSDoc : true;
```

Comment:
It seems strange to have an option that is `true` by default.

Could we change this option to be called `ignoreJSDoc` so that the default is `false` and people can override to `true`?

@mdjermanovic what do you think?

Reply:
Wouldn't the default for `ignoreJSDoc` be `true` as well?

If we want the default to be `false`, then perhaps `enforceForJSDoc` or `disallowJSDoc`? The default `disallowJSDoc: false` is a kind of double negation, so it might be confusing. But I think it's a good name since the wording allow/disallow seems suitable for this option because after converting to line comments it's no longer a valid JSDoc, so this option effectively does disallow multiline JSDoc comments.

Reply:
I used `allowJSDoc`  because I prefer positive booleans in general, indeed to avoid thinking in terms of double negatives. If you prefer to change it though, please confirm which one you would prefer as I see multiple possibilities above.

Reply:
Positive booleans are definitely preferable, however, not as a default value for an optional setting. Because undefined is falsy in JavaScript, its more idiomatic to have optional settings default to false.

Maybe `checkJSDoc` would work? So `checkJSDoc: false` is the default, so it leaves JSDoc comments alone, and `checkJSDoc: true` can override that and include JSDoc comments in the check?

Reply:
`checkJSDoc` sounds good to me.